modules/formulize/templates/admin/ui.html

```diff
@@ -160,6 +160,22 @@
     }
   });
 
+  $(".DCTemplates").click(function(){
+      var conf = confirm("Are you sure you want to Delete the template files? This should only be done when undergoing template development.");
+      if (conf) {
+          $.ajax({
+              url:'/formulize/deleteTempCache.php',

```

Comment:
Rather than linking directly to the file, the XOOPS_URL variable should be used because the path can change on different websites.